docs/scos/user/intro-to-spryker/releases/release-notes/release-notes-202302.0/security-release-notes-202302.0.md

```diff
@@ -0,0 +1,188 @@
+
+```bash
+composer require spryker/sales:"~11.30.2"
+composer show spryker/sales # Verify the version
+```
+
+When upgrading your `spryker/sales` module next time, consider selecting your next version according to the following guidelines.
+
+### Upgrade spryker/sales module versions, including 11.31.0 up to and including 11.33.x
+
+Complexity level: Low
+
+Update the `spryker/sales` module to the latest patch release of your minor and verify the version:
+
+```bash
+composer require spryker/sales:"~11.<your-minor>.<your-patch>"
+composer show spryker/sales # Verify the version
+```
+
+When upgrading your `spryker/sales` module next time, consider selecting your next version according to the following guidelines.
+
+### Upgrade spryker/sales module versions, including 11.34.0 up to and including 11.35.x
+
+Complexity level: Low
+
+Update the `spryker/sales` module to the latest patch release of your minor and verify the version:
+
+```bash
+composer require spryker/sales:"~11.<your-minor>.<your-patch>"
+composer show spryker/sales # Verify the version
+```
+
+When upgrading your `spryker/sales` module next time, consider selecting your next version according to the following guidelines. 
+
+### Upgrade on spryker/sales module versions 11.36.0 or later
+
+Complexity level: Low
+
+Update the `spryker/sales` module to the latest patch release of your minor and verify the version:
+
+```bash
+composer require spryker/sales:"~11.36.2"
+composer show spryker/sales # Verify the version
+```
+
+### Optional: Propel migration: Upgrade spryker/sales modules with versions 11.30.x or earlier to 11.31.2
+
+Complexity level: Low to medium
+
+{% info_block infoBox "Info" %}
+
+This involves additional migration effort because of a required upgrade of the Propel library.
+
+{% endinfo_block %}
+
+1. Update the `spryker/sales` version and verify the version.
+
+```bash
+composer require spryker/sales:"~11.31.2"
+composer show spryker/sales # Verify the version
+```
+
+2. Remove the `data/cache/propel/generated-conf/loadDatabase.php` file *manually* and regenerate the propel entities and query object:
+
+```bash
+rm data/cache/propel/generated-conf/loadDatabase.php
+ls data/cache/propel/generated-conf # Verify it was removed.
+vendor/bin/console propel:model:build
+ls data/cache/propel/generated-conf # Verify the file was created.
+```
+
+{% info_block warningBox "Attention" %}
+
+The following points are the potential BC breaking points from the Propel upgrade in 11.31.0:
+* Ensure any project code extending the Propel repository is updated (especially the method signatures).
+* Note that most Propel methods have parameter and return types being added. In case a project extension was not updated, you might encounter errors like this: `Fatal error: Declaration of ExtendedThing::process(array $items): array must be compatible with Thing::process(stdClass $item): array in ... on line ....`
```

Comment:
```suggestion
* Note that most Propel methods have parameter and return types added. In case a project extension was not updated, you might encounter errors like this: `Fatal error: Declaration of ExtendedThing::process(array $items): array must be compatible with Thing::process(stdClass $item): array in ... on line ....`
```